src/main/java/core/basesyntax/FileReaderImpl.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(String filePath) throws IOException {
```

Comment:
According to the checklist, it's recommended to use try-with-resources to ensure that the resources are closed after the program is finished with them. Since `Files.readAllLines` doesn't need an explicit close, it's fine here, but keep this in mind when dealing with other I/O operations.